benchmarks/src/main/java/zipkin/benchmarks/CodecBenchmarks.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015-2016 The OpenZipkin Authors
+ * Copyright 2015-2017 The OpenZipkin Authors
```

Comment:
most new lines of code here are actually increase in the benchmarks module. Otherwise, this change is pretty much line of code neutral